recipes/_restart_marker.rb

```diff
@@ -0,0 +1,13 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: _restart_marker
+#
+# Creates the restart file marker. This is used to ensure that the service is only restarted once and that it
+# only restarts when needed
+
+file 'splunk-marker' do
```

Comment:
was there a particular reason this resource was pulled out into a separate recipe? Was it just so we could unit test it easier?